### ISSUE TITLE
Extension flags Serverless templates as Cloudformation templates

### ISSUE THREAD

**Issue by mikepatrick** [awslabs/aws-cfn-lint-visual-studio-code#24]
First off, thanks for writing this and sharing it with us all.

Looking through the issue history, it looks like nailing down exactly which files to lint has been somewhat of a challenge.  I hate to throw another log on the fire, but here goes:

I use Serverless Framework for many of my AWS projects.  Since PR #22 this extension is flagging my Serverless templates as Cloudformation templates.

As a result, I now get lots of linting errors on the parts of my Serverless template that aren't valid Cloudformation:

![image](https://user-images.githubusercontent.com/4690013/46770918-f321e880-ccb6-11e8-80fd-ba62ccc99725.png)

I do deal with actual Cloudformation templates frequently enough that I'd like to continue using the extension.  It's also nice that the extension lints the Cloudformation generated by Serverless for me.  But I'd like it to go back to ignoring my `serverless.yml` files.

I'd be happy to dig in and raise a PR, but I'm not sure what the best way is to fix this for Serverless users.

- Additional heuristics to determine if a file is a serverless template?
- Just ignore files called `serverless.yml` altogether? (this convention is enforced by Serverless)
- Add a configuration to ignore certain user-specified files or file patterns?
- Something else?

Love the extension, and thanks again for putting it out here!

---

_Steps to reproduce:_

Open any valid serverless template that declares at least one resource in the `resources` section of the template.  For example:

```
 service: minimal-sls

 provider:
   name: aws
   runtime: nodejs8.10

 functions:
   helloWorld:
     handler: handler.helloWorld
     events:
       - http:
           path: hello-world
           method: get
 resources:
   Description: My stack description here
   Resources:
     # Cloudformation goes here
     MyBucket:
       Type: AWS::S3::Bucket
       Properties:
         BucketName: my-unique-bucket-name
```

**Comment by kddejong** [comment]
Sorry about that.  Working on something to not scan these templates automatically.  We get a lot of issues related to people not including the AWS Template Version which resulted in us trying to better detect a CloudFormation template.  We obviously missed on not flagging these as templates.  I'll work on a solution and try to get it released shortly.

**Comment by kddejong** [comment]
This should be resolved with v0.2.4

**Comment by mikepatrick** [comment]
Confirmed fixed for me.  Thanks for the rapid response @kddejong !
